packages/components/src/MultiSelect/MultiSelect.css

```diff
@@ -0,0 +1,38 @@
+.multiSelectContainer {
+  position: relative;
+  width: 100%;
+}
+
+.multiSelect {
+  display: flex;
+  padding-top: calc(var(--space-base) - var(--space-smallest));
+  padding-bottom: calc(var(--space-base) - var(--space-smallest));
+  padding-left: var(--space-base);
+  padding-right: var(--space-base);
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-base);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.multiSelect p {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.multiSelectContainer:hover,
+.multiSelectContainer.multiSelect {
```

Comment:
Do we still need these input-specific stylings since the component no longer uses a FormField internally?

Reply:
yes, without it I can still see the blinking cursor in front of the text even though it is not an input text anymore